openadmet/models/eval/binary.py

```diff
@@ -0,0 +1,72 @@
+from sklearn.metrics import precision_recall_curve, auc
+import matplotlib.pyplot as plt
+import numpy as np
+import wandb
+import pandas as pd
+
+from openadmet.models.eval.eval_base import EvalBase, evaluators
+
+
+@evaluators.register("PosthocBinaryMetrics")
+class PosthocBinaryMetrics(EvalBase):
+
+    """
+    Intended to be used for regression-based models to calculate
+    precision and recall metrics for user-input
+
+    Not intended for binary models
+    """
+
+    def evaluate(self, y_true=None, y_pred=None, cutoffs=None, report=False, output_dir=None, **kwargs):
```

Comment:
Use type hints to suggest what type each arg / kwarg is, makes easier to follow.